generated/nifgen/session.py

```diff
@@ -3619,6 +3621,8 @@ def create_freq_list(self, waveform, frequency_array, duration_array):
         vi_ctype = _visatype.ViSession(self._vi)  # case S110
         waveform_ctype = _visatype.ViInt32(waveform.value)  # case S130
         frequency_list_length_ctype = _visatype.ViInt32(0 if frequency_array is None else len(frequency_array))  # case S160
+        if duration_array is not None and len(duration_array) != len(frequency_array):  # case S160
```

Comment:
Looks like a small bugfix. Worth noting?

Reply:
I don't think we need to.